collect_app/src/main/java/org/odk/collect/android/openrosa/api/OpenRosaFormListApi.java

```diff
@@ -27,23 +30,43 @@
     private final String serverURL;
     private final String formListPath;
 
-    public OpenRosaFormListApi(OpenRosaXmlFetcher openRosaXMLFetcher, String serverURL, String formListPath) {
-        this.openRosaXMLFetcher = openRosaXMLFetcher;
+    public OpenRosaFormListApi(String serverURL, String formListPath, OpenRosaHttpInterface openRosaHttpInterface, WebCredentialsUtils webCredentialsUtils) {
```

Comment:
I killed all the error messages coming out of here and pretty much all the error logging. Most of it isn't needed to show the errors we're trying to show. Good to check and see if any of it feels like something we'd need to keep.

Reply:
Yup, looks good to me.